ktor-http/common/src/io/ktor/http/RequestConnectionPoint.kt

```diff
@@ -23,13 +23,40 @@ public interface RequestConnectionPoint {
     /**
      * Request port, for example 80 or 443
      */
+    @Deprecated("Use localPort or serverPort instead")
     public val port: Int
 
+    /**
+     * Request port on which the request was received, for example 80 or 443
```

Comment:
I'm not sure that I understand the doc. I would say it's `The port used to accept the client connection. It's usually specified in the client request`

Reply:
I took the same terminology and naming as in Jetty